tests/5.0/target_teams_distribute/test_target_teams_distribute_reduction_or.c

```diff
@@ -0,0 +1,76 @@
+//===--- test_target_teams_distribute_reduction_or.c-------------------------===//
+//
+// OpenMP API Version 4.5 Nov 2015
+//
+// This test uses the reduction clause on a target teams distribute directive,
+// testing that the variable in the reduction clause is properly reduced using
+// the or operator.
+//
+////===----------------------------------------------------------------------===//
+
+#include <omp.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include "ompvv.h"
+#include <math.h>
+
+#define N 1024
+
+int test_or() {
+  char a[N];
+  double false_margin = pow(exp(1), log(.5)/N);   // See the 'and' operator test for
```

Comment:
Shouldn't this be the other way. We want to have the chance to test all zeros to obtain a false result for or